stubs/extensions/soap.phpstub

```diff
@@ -1,5 +1,97 @@
 <?php
 
+<?php
```

Comment:
There's a duplicated php tag

Reply:
Whoops! That was careless! I've gotten too used to Psalm catching all my boneheaded mistakes. Thanks!